### PR TITLE
Standardised Autocompletes on react-select

### PR DESCRIPTION
**Pull request by AlexGilleran**
### What this PR does

Doesn't fix anything yet - it's a precursor to #2334 

This standardises all the autocompletes up until (inclusive of) the affiliated organization one.

So:
#### Keywords
image

#### Themes
image

#### Publishing Org
image

#### Team
image

#### Affiliated Org
image

### Checklist

-   [x] There are unit tests to verify my changes are correct or unit tests aren't applicable
-   [x] I don't think changes is really needed given that this is just making it do what it already does but neater
-   [x] No issue

### CODE REVIEW

File: magda-web-client/src/Components/Dataset/Add/Pages/People/OrgUnitDropdown.tsx [hunk 1]
```diff
@@ -38,29 +41,31 @@ export default function OrgUnitDropdown({
...
+            <Select
+                className="react-select"
+                isMulti={false}
+                isSearchable={false}
+                onChange={(rawValue, action) => {
+                    const value = rawValue as (
+                        | { value: string }
+                        | undefined
+                        | null);
+                    console.log(value);
```
Comment (t83714) [inline magda-web-client/src/Components/Dataset/Add/Pages/People/OrgUnitDropdown.tsx]: the debug code to be removed

File: magda-web-client/src/Components/Dataset/Add/Pages/People/OrganisationAutocomplete.tsx [hunk 2]
```diff
@@ -1,46 +1,90 @@
...
+    return {
+        existingId: choice.existingId,
+        value: choice.existingId || choice.name,
+        label: choice.name
+    };
 }
 
 export default function OrganisationAutocomplete(props: Props) {
-    const { onOrgSelected, defaultValue } = props;
+    console.log(props);
```
Comment (t83714) [inline magda-web-client/src/Components/Dataset/Add/Pages/People/OrganisationAutocomplete.tsx]: debug code to be removed

**Comment by AlexGilleran** [comment]
Oops! @t83714 updated :)

File: magda-web-client/src/Components/Dataset/Add/Pages/People/OrgUnitDropdown.tsx (hunk 1 above)
Comment (t83714) [reply magda-web-client/src/Components/Dataset/Add/Pages/People/OrgUnitDropdown.tsx]: @AlexGilleran It seems this `console.log` (in `Components/Dataset/Add/Pages/People/OrgUnitDropdown.tsx`) still there

**Pull request merged**